src/components/Confirmation/Close/Close.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './Close.module.css';
+
+const Close = (props) => {
```

Comment:
It is not a good idea to have a separated component. In general, it will be Button component with some specific styling